qutip/bloch.py

```diff
@@ -284,16 +284,24 @@ def __str__(self):
         return s
 
     def _repr_png_(self):
-        from IPython.core.pylabtools import print_figure
+        try:
+            from IPython.core.pylabtools import print_figure
+        except ImportError:
+            return None
```

Comment:
I would prefer not to have these `try ... except ...`  and instead just let the import error propagate. I would expect only IPython will call `_repr_png_` and `_repr_svg` so if IPython isn't installed, something strange has happened. Returning `None` just silences the error.